### ISSUE TITLE
Consistent Websocket errors

### ISSUE THREAD

**Issue by eltiel** [gotify/server#273]
**Can the issue be reproduced with the latest available release? (y/n)**
Y

**Which one is the environment gotify server is running in?**
- [x] Docker
- [ ] Linux machine
- [ ] Windows machine
<details><summary>Docker startup command or config file here (please mask sensitive information)</summary><br>

```
N/A - Synology Docker
```
</details>

**Do you have an reverse proxy installed in front of gotify server? (Please select None if the problem can be reproduced without the presense of a reverse proxy)**
- [x] None
- [ ] Nginx
- [ ] Apache
- [ ] Caddy
<details><summary>Reverse proxy configuration (please mask sensitive information)</summary><br>

```
Happened with nginx as reverse proxy, but reproducable even when connected directly
```
</details>

**On which client do you experience problems? (Select as many as you can see)**
- [ ] WebUI
- [ ] gotify-cli
- [x] Android Client 
- [ ] 3rd-party API call (Please include your code)

**What did you do?**
I left the app running. Screen did not even turn off and I kept refreshing the logs. Every few minutes, a Websocket error appears

**What did you expect to see?**
No errors and remain connected

**What did you see instead? (Include screenshots, android logcat/request dumps if possible)**
![Screenshot_2020-02-18-22-35-08-360_com github gotify](https://user-images.githubusercontent.com/11868456/74746614-abdf3b00-52a0-11ea-8fa9-bd61d4d97bb1.jpg)

**Comment by eltiel** [comment]
This problem is pretty much identical to https://github.com/gotify/server/issues/217

**Comment by eltiel** [comment]
More logs:

![Screenshot_2020-02-18-22-51-28-087_com github gotify](https://user-images.githubusercontent.com/11868456/74747206-830b7580-52a1-11ea-9ad1-07435c218548.jpg)

**Comment by jmattheis** [comment]
The connection always breaks after 60seconds. It seems like a timeout which cancels the websocket connection. This is most likely not the fault of Gotify but another services that acts as a proxy. With https://github.com/gotify/server/pull/261 the ping interval from our websocket got reduced to less than 60s. This could fix the problem, could you try the newly released v2.0.14 which includes https://github.com/gotify/server/pull/261?

**Comment by eltiel** [comment]
Appears to be working fine with 2.0.14. Has been about 20 minutes without any errors.

Thanks for this excellent piece of software.
